staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/time_proto.go

```diff
@@ -90,3 +90,10 @@ func (m *Time) MarshalTo(data []byte) (int, error) {
 	}
 	return m.ProtoTime().MarshalTo(data)
 }
+
+func (m *Time) MarshalToSizedBuffer(data []byte) (int, error) {
```

Comment:
Add a comment please

Reply:
Comment is wrong?

Reply:
good point - but I think it's better to a send a followup PR for it :)